.github/workflows/ci.yml

```diff
@@ -136,3 +136,33 @@ jobs:
       - name: Dump docker logs on failure
         if: failure()
         uses: jwalton/gh-docker-logs@v1
+
+  publish:
+    if: |
+      github.ref == 'refs/heads/main' ||
+      github.ref == 'refs/heads/develop'
+    runs-on: ubuntu-latest
+    steps:
+      # Only if all other jobs pass.
+      - name: Wait on tests
+        uses: lewagon/wait-on-check-action@v0.2
+        with:
+          ref: ${{ github.ref }}
+          running-workflow-name: 'publish' #this job shouldn't wait for itself
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 20 # seconds
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Install only toml, which is used to extract version.
+      # We don't do anything else with Python in this job.
+      - name: Install toml
+        run: python -m pip install toml
+
+      - name: Build & Publish Images
+        shell: bash
+        run: ./.github/workflows/publish_to_docker.sh
```

Comment:
this works for now, but we use the GH action script for this in SEED: https://github.com/SEED-platform/seed/blob/develop/.github/workflows/publish.yml#L63-L70. Just an idea for the future.

Reply:
Thanks!  We build and push two containers based on a bunch of config in the docker-compose, so i didn't see a straightforward action for this.